src/main/java/blackjack/model/participant/Dealer.java

```diff
@@ -0,0 +1,31 @@
+package blackjack.model.participant;
+
+import blackjack.model.card.Card;
+import blackjack.model.card.CardDeck;
+import blackjack.model.state.DealerDrawState;
+import blackjack.model.state.DrawState;
+import blackjack.model.state.State;
+
+public class Dealer extends Participant {
+
+    private static final int DEALER_FIRST_CARD = 0;
+    private static final String DEALER_NAME = "딜러";
+
+    public Dealer(State currentState) {
+        super(new Name(DEALER_NAME), currentState);
+    }
+
+    @Override
+    public void play(CardDeck cardDeck) {
+        this.currentState = currentState.draw(cardDeck);
+        if (currentState instanceof DrawState) {
+            this.currentState = ((DrawState) currentState).turnDealerDrawState();
+            this.currentState = ((DealerDrawState)currentState).checkStandOrBustState();
+        }
+    }
+
+    public Card getFirstCard(){
```

Comment:
```suggestion
    public Card getFirstCard() {
```
포매팅이 잘못 되어 있네요. 위의 형 변환도 마찬가지입니다.

`this.currentState = ((DealerDrawState) currentState).checkStandOrBustState();`